### PR TITLE
Feat/invite mailer

### PR DESCRIPTION
**Pull request by cmehrabian**
Make sure you update your development.json

### CODE REVIEW

File: server/views/email-template.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+module.exports.emailTemplate = function(link) {
+  return (`<body bgcolor="#f6f6f6" style="font-family: Arial; background-color: #f6f6f6;">
+  <table width="630" class="container" align="center" cellpadding="0" cellspacing="0">
+  <tr>
+    <td>
+      <table align="left">
+        <tr>
+          <td width="188" class="Logo">
+            <img src="http://www.rickygipson.com/images/codepen/company_logo.png">
```
Comment (clarissachin) [inline server/views/email-template.js]: When trying to pull up this image, it says: "This site is temporarily unavailable" (?)

Comment (cmehrabian) [reply server/views/email-template.js]: Ah yes! Lets replace it with any mock brand

Comment (kolya182) [reply server/views/email-template.js]: More likely it was deleted from the server. Try upload to AWS and replace the link.

File: db/migrations/20170326215143_initial.js [hunk 2]
```diff
@@ -1,14 +1,35 @@
...
+    }),
     knex.schema.createTableIfNotExists('profiles', function (table) {
       table.increments('id').unsigned().primary();
       table.string('first', 100).nullable();
       table.string('last', 100).nullable();
       table.string('display', 100).nullable();
       table.string('email', 100).nullable().unique();
       table.string('phone', 100).nullable();
-      table.string('avatar', 500).nullable();
+      table.string('avatar', 300).nullable();
```
Comment (clarissachin) [inline db/migrations/20170326215143_initial.js]: Keke, my update to 500 might've been too much ;) 300 is good. Thanks!!!

File: db/migrations/20170326215143_initial.js [hunk 3]
```diff
@@ -1,14 +1,35 @@
...
+      table.string('type', 20).nullable();
+      table.string('title');
+      table.string('description');
+      table.string('etag');
+      table.timestamp('created_at').defaultTo(knex.fn.now());
+      table.timestamp('updated_at').defaultTo(knex.fn.now());
+    }),
+    knex.schema.createTableIfNotExists('profiles_families', (table) => {
+      table.increments('id').unsigned().primary();
+      table.integer('familie_id').references('families.id').onDelete('CASCADE');
```
Comment (clarissachin) [inline db/migrations/20170326215143_initial.js]: familie_id ==> families_id ... ?

Comment (cmehrabian) [reply db/migrations/20170326215143_initial.js]: I know, weird right? Bookshelf likes assuming join table comparisons by dropping the 's' in plural sounding models.

**Pull request merged**